Source/SwiftLintFramework/Rules/IdentifierNameRule.swift

```diff
@@ -35,7 +35,7 @@ public struct IdentifierNameRule: ASTRule, ConfigurationProviderRule {
                 return []
             }
 
-            let isFunction = SwiftDeclarationKind.functionKinds.contains(kind)
+            let isFunction = SwiftDeclarationKind.functionKinds.contains(kind) || kind == .enumelement
```

Comment:
Maybe for better naming and understand you could separate in two variables. Like:

```swift
let isFunction = SwiftDeclarationKind.functionKinds.contains(kind)
let isEnumelement = kind == .enumelement
```

Or jus update the `isFunction` to something like `isFunctionOrEnumelement`.